tests/integration/api/test_api.py

```diff
@@ -258,147 +157,93 @@ def test_timeboard(self):
-            (int(time.mktime((now - datetime.timedelta(minutes=17)).timetuple())), 5),
-            (int(time.mktime((now - datetime.timedelta(minutes=17)).timetuple())) + 1, 15),
-            (int(time.mktime((now - datetime.timedelta(minutes=15)).timetuple())), 15),
-            (int(time.mktime((now - datetime.timedelta(minutes=15)).timetuple())) + 1, 5),
-            (int(time.mktime((now - datetime.timedelta(minutes=14)).timetuple())), 5),
-            (int(time.mktime((now - datetime.timedelta(minutes=14)).timetuple())) + 1, 15),
-            (int(time.mktime((now - datetime.timedelta(minutes=12)).timetuple())), 15),
-            (int(time.mktime((now - datetime.timedelta(minutes=12)).timetuple())) + 1, 5),
-            (int(time.mktime((now - datetime.timedelta(minutes=11)).timetuple())), 5),
-        ]
-
-        dog.Metric.send(metric='matt.metric', points=matt_series, host="matt.metric.host")
+            metric_query_single = dog.Metric.query(
+                start=now_ts - 600,
+                end=now_ts + 600,
+                query="{}{{host:{}}}".format(metric_name_single, host_name)
+            )
+            metric_query_multi = dog.Metric.query(
+                start=now_ts - 600,
+                end=now_ts + 600,
+                query="{}{{host:{}}}".format(metric_name_multi, host_name)
+            )
+        print(retry_time)
+        print(metric_query_single)
+        print(metric_query_multi)
+        assert len(metric_query_single["series"]) == 1
+        assert metric_query_single["series"][0]["metric"] == metric_name_single
+        assert metric_query_single["series"][0]["scope"] == "host:{}".format(host_name)
+        assert len(metric_query_single["series"][0]["pointlist"]) == 1
+        assert metric_query_single["series"][0]["pointlist"][0][1] == 1
+
+        assert len(metric_query_multi["series"]) == 1
+        assert metric_query_multi["series"][0]["metric"] == metric_name_multi
+        assert metric_query_multi["series"][0]["scope"] == "host:{}".format(host_name)
+        assert len(metric_query_multi["series"][0]["pointlist"]) == 2
+        assert metric_query_multi["series"][0]["pointlist"][0][1] == 1
+        assert metric_query_multi["series"][0]["pointlist"][1][1] == 2
 
     def test_type_check(self):
-        dog.Metric.send(metric="test.metric", points=[(time.time() - 3600, 1.0)])
-        dog.Metric.send(metric="test.metric", points=1.0)
-        dog.Metric.send(metric="test.metric", points=(time.time(), 1.0))
-
-    @pytest.mark.monitor
-    def test_monitors(self):
-        query = "avg(last_1h):sum:system.net.bytes_rcvd{host:host0} > 100"
+        assert dog.Metric.send(metric="test.metric", points=[(time.time() - 3600, 1.0)])["status"] == "ok"
+        assert dog.Metric.send(metric="test.metric", points=1.0)["status"] == "ok"
+        assert dog.Metric.send(metric="test.metric", points=(time.time(), 1.0))["status"] == "ok"
```

Comment:
What are we testing here? Should we try to query as well?

Reply:
testing the different types you can pass to the `points`. Doing that as part of the test_metrics instead, and removing that one